app/javascript/react/components/MapButtons/MapButtons.tsx

```diff
@@ -76,7 +76,8 @@ const MapButtons = () => {
           isActive={activeButton === ButtonTypes.SHARE}
         />
       </S.MapButtons>
-      {showFilters && <DesktopSessionFilters />}
+      {/* {showFilters && <DesktopSessionFilters />} */}
```

Comment:
Should we keep this commented line?

Reply:
Commented line is the correct one. I'll remove the other one